src/plugins/stylesheet/SassPlugin.ts

```diff
@@ -80,12 +80,20 @@ export class SassPluginClass implements Plugin {
                 if (/https?:/.test(url)) {
                     return done({ url });
                 }
+
                 for (let key in options.macros) {
                     if (options.macros.hasOwnProperty(key)) {
                         url = url.replace(key, options.macros[key]);
                     }
                 }
-                done({ file: path.normalize(url) });
```

Comment:
Why did you remove the normalization?

Reply:
It's not been removed - it does it still, see line 90. It normalises then it checks for overrides 👍